libtiledbvcf/src/CMakeLists.txt

```diff
@@ -179,7 +202,8 @@ target_link_libraries(tiledbvcf-bin
     CLI11::CLI11
     HTSlib::HTSlib
     TileDB::tiledb_shared
-    spdlog::spdlog
+# TBD: removing this avoids msvc/windows spdlog build issues, why is it here, why does removing this help?
```

Comment:
Please make this and the other one conditional on !WIN32 and add a _specific_, clear comment on what the build issues are, or if more than 2-3 lines, a reference to a shortcut story for more information.